include/ideep/operators/inner_product.hpp

```diff
@@ -499,7 +510,7 @@ struct inner_product_forward
     args.insert({DNNL_ARG_SCRATCHPAD, scratchpad});
     args.insert(
         {DNNL_ARG_ATTR_MULTIPLE_POST_OP(0) | DNNL_ARG_SRC_1, expected_other});
-    if (reorder_src) {
+    if (reorder_src) { // reorder src
```

Comment:
keep "// reorder src" comment at the "else" clause?

Reply:
changed.